javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/ignore/GlobalIgnoredTypesConfigurer.java

```diff
@@ -110,7 +110,9 @@ private static void configureIgnoredTypes(IgnoredTypesBuilder builder) {
 
   private static void configureIgnoredClassLoaders(IgnoredTypesBuilder builder) {
     builder
+        .ignoreClassLoader("groovy.lang.GroovyClassLoader")
```

Comment:
Do you know when/in what cases the `GroovyClassLoader` is used? Looks like all our groovy tests are passing, which probably means it's not being used there. Does excluding it here mean that all groovy apps (like e.g. grails apps) will no longer be instrumented?

Reply:
I think it is used when the groovy script is compiled on the fly.

Reply:
> Do you know when/in what cases the GroovyClassLoader is used?

I'm not sure of all the cases, but in the sample code above it is used. @laurit is probably right here.